### ISSUE TITLE
Data Issue:  Days with 0 cases not incorporated properly

### ISSUE THREAD

**Issue by eckurilla** [nytimes/covid-19-data#591]
Using Fairfax County, VA as an example: On 5/25 there were 0 reported cases. It appears as though this day is now omitted from the averages instead of incorporated as a 0 case day.

**Comment by ddenenberg71** [comment]
I think you meant Fairfax _City_ (51600) not Fairfax _County_ (51059).  Your comment just caught my eye since I'm a resident of Fairfax County (and creator of covid19va.net).

**Comment by eckurilla** [comment]
Oh! Perhaps we found the issue. The NYTimes Fairfax _County_ page is showing 0 cases for 5/25, but the averages certainly don't seem to incorporate a 0 day. (I admit I did not run the calcs myself). Maybe there is a discrepancy between the data being used for the averages & what is being displayed in the daily case numbers for 5/25.  
https://www.nytimes.com/interactive/2021/us/fairfax-virginia-covid-cases.html
